lib/cmd/fh3/admin/users/changeauthpolicies.js

```diff
@@ -1,8 +1,9 @@
 /* globals i18n */
 var fhc = require("../../../../fhc");
 
+//@Deprecated;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ACommentedCode)